src/lwc/utilInput/utilInput.js

```diff
@@ -16,11 +16,15 @@ const DATETIME = 'datetime-local';
 const YES = 'Yes';
 const TEXT = 'text';
 const TRUE = 'true';
+const RICH_TEXT_FORMATS = [
```

Comment:
What is this being used for? It doesn't seem like this + richTextFormats is actually being used, unless I'm missing something?

Reply:
Ah, yes this is the whitelist of formats being set in the `lightning-input-rich-text` component's `formats` property. That property controls some of the buttons and what formatted content can be pasted into that rich input.

I straight up lifted this list from the one being used in the renderer so we have alignment across Gift Entry with regards to rich text inputs.